docs/src/modules/components/MarkdownElement.js

```diff
@@ -601,12 +608,15 @@ const Root = styled('div')(
         '&:hover .anchor-link, & .comment-link': {
           color: `var(--muidocs-palette-primary-300, ${darkTheme.palette.primaryDark[300]})`,
           borderColor: `var(--muidocs-palette-divider, ${darkTheme.palette.divider})`,
-          backgroundColor: alpha(darkTheme.palette.primary[900], 0.3),
+          backgroundColor: alpha(darkTheme.palette.primaryDark[700], 0.5),
           '&:hover': {
             borderColor: `var(--muidocs-palette-primary-900, ${darkTheme.palette.primary[900]})`,
             backgroundColor: alpha(darkTheme.palette.primary[900], 0.6),
             color: `var(--muidocs-palette-primary-100, ${darkTheme.palette.primary[100]})`,
           },
+          '&:focus-visible': {
+            outline: `3px solid ${alpha(lightTheme.palette.primary[500], 0.5)}`,
```

Comment:
I'm a bit surprised by a `lightTheme` in the definition of the dark theme colors

Reply:
Oops...